site/source/user-manual/flashing/flashing-pi.rst

```diff
@@ -21,53 +21,34 @@ Select your OS to continue:
 
     .. group-tab:: Linux
 
-        #. Open a terminal in the folder you downloaded to and extract with:
+        (Optional, but recommended) Verify the checksum against the one listed on GitHub (SHA256):
         
             .. code-block::
             
-                tar -xzvf startos-0.3.4.2-bbd66e9-20230519_raspberrypi.img.gz
-
-        #. (Optional, but recommended) Verify the checksum against the one listed on GitHub (SHA256):
-        
-            .. code-block::
-            
-                sha256sum startos-0.3.4.2-bbd66e9-20230519_raspberrypi.img
-
-            or (BLAKE3):
-
-            .. code-block::
-            
-                b3sum startos-0.3.4.2-bbd66e9-20230519_raspberrypi.img
+                sha256sum startos-0.3.4.2-efc56c0-20230525_raspberrypi.img.gz
         
     .. group-tab:: Mac
             
-        #. Right-click ``startos-0.3.4.2-bbd66e9-20230519_raspberrypi.img.gz``, click "Open with," then click Archive Utility to extract.
-
-        #. (Optional, but recommended) Verify the checksum against the one listed on GitHub by opening a terminal and entering:
+        (Optional, but recommended) Verify the checksum against the one listed on GitHub by opening a terminal and entering:
 
             .. code-block::
 
-                openssl dgst -sha256 startos-0.3.4.2-bbd66e9-20230519_raspberrypi.img
+                openssl dgst -sha256 startos-0.3.4.2-efc56c0-20230525_raspberrypi.img.gz
 
     .. group-tab:: Windows
-            
-        #. Open a ``cmd`` prompt "As Administrator" in the folder you saved the image to and use the following command to extract it:
-
-            .. code-block::
-            
-                tar -xzvf startos-0.3.4.2-bbd66e9-20230519_raspberrypi.img.gz
-
-        #. (Optional, but recommended) Verify the checksum against the one listed on GitHub by running:
+        
+        (Optional, but recommended) Verify the checksum against the one listed on GitHub by opening Windows PowerShell, changing your directory to where you downloaded the .img.gz file, and running `Get-FileHash`:
```

Comment:
Tested this, and neither cmd (nor cmd run as administrator) are enough, you have to be in powershell for Get-FileHash to be an available 'cmdlet'